index.html

```diff
@@ -9,7 +9,7 @@
   <h1 class="text-center index-title">
     <span class="img-index-title"></span>
     <span class="img-index-title-mobile">Super Simple WYSIWYG<br>Editor on Bootstrap</span>
-    <a href="{{ site.repository }}/archive/master.zip" class="index-download">
+    <a href="{{ site.repository }}/releases/download/v0.8.18/summernote-0.8.18-dist.zip" class="index-download">
```

Comment:
I would like to change it as below.
```suggestion
    "{{ site.repository }}/releases/download/v{{ site.version }}/summernote-{{ site.version }}-dist.zip"
```

Reply:
Good comment.